README.md

```diff
@@ -28,19 +28,18 @@ Current version of the [pipeline](https://github.com/phenoscape/phenoscape-owl-t
 ***
 
 ## Deployment
+1. Download docker image `phenoscape/pipeline-tools` from [here](https://hub.docker.com/r/phenoscape/pipeline-tools).
+2. Execute `run.sh` to launch the container. Pass in commands to build different targets as arguments to `run.sh`. Pass `make all` to run the entire pipeline. 
+3. `run.sh` also launches a `bash` shell in the container to interactively work with the pipeline. 
```

Comment:
I'm not sure this is true. It just runs whatever command you pass as an argument.

Reply:
No, I have run this script without passing in any arguments, and it just straight launches the bash shell.